flink-yarn/src/main/java/org/apache/flink/yarn/configuration/YarnConfigOptions.java

```diff
@@ -171,6 +171,19 @@
 				" setting and use cluster default priority. Please refer to YARN's official documentation for specific" +
 				" settings required to enable priority scheduling for the targeted YARN version.");
 
+	/**
+	 * Yarn session client uploads flink jar and user libs to file system (hdfs/s3) as local resource for yarn
+	 * application context. The replication number changes the how many replica of each of these files in hdfs/s3.
+	 * It is useful to accelerate this container bootstrap, when a Flink application needs more than one hundred
+	 * of containers. If it is configured, Flink will use the default replication value in hadoop configuration.
```

Comment:
"If it is configured" -> "If it is not configured"

Reply:
Done.